### ISSUE TITLE
Why get_score() function use parameter is starcoder_model than starcoder_model_ref ? 

### ISSUE THREAD

**Issue by Yi-Qi638** [ashishjamarkattel/reinforment-learning-with-human-feedback#6]
In the notebook rhlf.ipynb

logits = get_score(starcoder_model,starcoder_tokenizer, texts)
rewards = pos_logit_to_reward(logits, task_list)

Since these rewards are derived from the logits, should I utilize starcoder_model_ref to obtain the scores instead？

**Comment by ashishjamarkattel** [comment]
You will be training, Your policy model, with the help of reward model, So in this case reward model is constant. That is why you use starcoder_model ie policy model
